environment/swarmball_env.py

```diff
@@ -4,17 +4,24 @@
 from .simulation.simulation import SwarmBallSimulation
 
 class SwarmBall(gym.Env):
-    def __init__(self):
-        self.sim = SwarmBallSimulation()
+    def __init__(self, acc_factor=0.25, thresh_amount=3, **kwargs):
+        self.sim = SwarmBallSimulation(**kwargs)
+        self.thresholds = [[] for _ in range(thresh_amount)]
+        self.thresh_vel = [0,0,0]
+        self.acc_factor = acc_factor
 
-    def seed(self, seed=None):
-        """
-            To przerażające nasiono!!!
-        """
+    def reward(self):
         pass
 
     def step(self, action):
+        self.thresholds.pop(0)
```

Comment:
Dlaczego nie `thresholds[0] = self.sim.threshold_positions()` zamiast tych 2 linijek?

Reply:
Bo to kolejka fifo jest.

Reply:
To czegoś innego tu nie rozumiem - dlaczego tu wgl jest kolejka? W sensie co ona nam tu daje, czy w każdym stepie ona nie będzie miała długości 1?

Reply:
Wygląda, że zwracane są nam pozycje thresholdow z 3 ostatnich kroków, więc się chyba znowu nie dogadaliśmy xd  zdjęcia mapy miały być z 3 ostatnich ale to składowane jest w kolejce po stronie sieci, dlatego wystarczy nam tylko jedno aktualne zawsze. Więc nawet jeśli chcieli byśmy 3 osatatnie zestawy thresholdow, to dla porządku i tak powinny być w kolejce w sieci, a zwracane nam tylko te jedne aktualne

Reply:
a bieżący plan był taki, że sieć łyka 3 zdjęcia  mapy i 1 pozycje thresholdów (chociaż w sumie z astanawiałem się nad przynajmniej dwoma, ale takie rzeczy to juz przy tunowaniu i po stronie sieci)

Reply:
Nie pisz tu co łyka sieć, bo to znowu jest po stronie sieci a nie tutaj, więc tylko miesza XD

Reply:
Oooo.. Dobra, juz czaje skad "mogą" brać się nieporozumia, musze sie zacząć jaśniej wyrażać xd

Reply:
A więc: od enva chcemy aktualne 1 zdjecie mapy oraz 1 wektor aktualnych thresholdow